src/material/paginator/paginator.html

```diff
@@ -14,7 +14,7 @@
             <mat-select
               [value]="pageSize"
               [disabled]="disabled"
-              [aria-labelledby]="_pageSizeLabelId"
+              [aria-label]="_intl.itemsPerPageLabel + pageSize"
```

Comment:
These can't just be concatenated together- [go/concat-considered-harmful](https://goto.google.com/concat-considered-harmful)